webhook_server_container/tests/test_github_api.py

```diff
@@ -21,7 +31,15 @@ def __init__(self, path: str):
     @property
     def tree(self):
         trees = []
-        for _path in ["OWNERS", "test1/OWNERS", "code/file.py", "README.md"]:
+        for _path in [
+            "OWNERS",
+            "folder1/OWNERS",
```

Comment:
Do you need to update it? Between the tests and get_contents() I was a bit confused.